modules/virt-adding-vm-to-service-mesh.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-connecting-vm-to-service-mesh.adoc
+
+[id="virt-adding-vm-to-service-mesh_{context}"]
+= Configuring a virtual machine for the service mesh
+
+To add a virtual machine (VM) workload to a service mesh, enable automatic sidecar injection in the VM configuration file by setting the `sidecar.istio.io/inject` annotation to `true`. Then expose your VM as a service to view your application in the mesh.
+
+
+.Procedure
+
+. Edit the VM configuration file to add the `sidecar.istio.io/inject: "true"` annotation.
++
+.Example configuration file
+[source,yaml]
+----
+  apiVersion: kubevirt.io/v1
+  kind: VirtualMachine
+  metadata:
+    labels:
+      kubevirt.io/vm: vm-istio
+    name: vm-istio
+  spec:
+    running: false
```

Comment:
OCP Virtualization documentation seems to prefer `RunStrategy` over `running` https://docs.openshift.com/container-platform/4.8/virt/virtual_machines/virt-create-vms.html#virt-about-runstrategies-vms_virt-create-vms

I guess it would better to use that then:
```
  apiVersion: kubevirt.io/v1
  kind: VirtualMachine
  metadata:
    labels:
      kubevirt.io/vm: vm-istio
    name: vm-istio
  spec:
    runStrategy: Always
```

Reply:
Changed the field to `runStrategy: Always`